internal/server/singleprocess/state/server_apitoken.go

```diff
@@ -0,0 +1,27 @@
+package state
+
+import (
+	"github.com/boltdb/bolt"
+)
+
+var (
+	serverAPIToken = []byte("api_token")
```

Comment:
Recommend renaming this and all the function calls to `url_token` and `ServerURLTokenSet` since `api_token` is relatively vague and somewhat confusing cause we also ave API tokens for Waypoint itself and this is unrelated.